src/main/resources/assets/tanukidecor/models/block/alarm_clock/alarm_clock.json

```diff
@@ -2,107 +2,498 @@
   "credit": "Made by Skart2000",
   "parent": "minecraft:block/block",
   "render_type": "minecraft:cutout",
-  "texture_size": [32, 32],
+  "texture_size": [
```

Comment:
Please revert all formatting changes to JSON files in the `models` folder. The changes make the files longer and more difficult to read. I very intentionally had things like in-line arrays and in-line object definitions to make it easier for me to work with them.